### PR TITLE
Added clear filter in secondary navbar

### PR DESCRIPTION
**Pull request by bereketsega**
This PR adds clear all filter functionality and updates the selected filters color in the secondary navbar of the cc dashboard.

## Demo

https://github.com/Cyber4All/competency-client/assets/72763770/2f09363f-aace-4e5a-b5ee-676859ea2eaa

### CODE REVIEW

**Review by pmanal1 — APPROVED** [review]
Looks great. I would suggest to conditionally display the "clear filters" button when there are filters added

**Review by mfranz1 — COMMENTED** [review]
the clear filters button should toggle on and off. It should only display when filters are applied. Also, I think you may have overlapping PRs? I don't see where you added the html for  the clear filters button

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Comment by bereketsega** [comment]
> the clear filters button should toggle on and off. It should only display when filters are applied. Also, I think you may have overlapping PRs? I don't see where you added the html for the clear filters button
The clear filters button toggle is added now.
The button to clear all filter is added [here](https://github.com/Cyber4All/competency-client/pull/109/files#diff-7e5d46918e48c28545ab9cd26f142c803290123e513bac4c70be94583a0e0db1R10).

File: src/app/shared/components/filtering-dropdowns/filtering-dropdowns.component.ts [hunk 1]
```diff
@@ -96,7 +107,7 @@ export class FilteringDropdownsComponent implements OnInit {
   }
 
   calculateStyles(item: any) {
-    return this.selectedItems.includes(item) ? { 'color': '#376ED6' } : { 'color': '#454545' };
+    return this.selectedItems.includes(item.id) ? { 'color': '#376ED6' } : { 'color': '#454545' };
```
[the remark this answers is not in the document]
Comment (bereketsega) [reply src/app/shared/components/filtering-dropdowns/filtering-dropdowns.component.ts]: Used `blue` and `black` to refer to the class and used the class in `.scss` to change the color using values from our `vars.scss`. Please let me know if this fulfills the requirements.

**Comment by bereketsega** [comment]
## New Demo

https://github.com/Cyber4All/competency-client/assets/72763770/10b824b0-dcfb-415b-96a8-c835ac676207

**Review by mfranz1 — APPROVED** [review]
Looks good, I asked Katelyn to take a look at how you implemented your styles.

**Review by katieblomquist — APPROVED** [review]
Looks good to me!

**Pull request merged**